integration_tests/test_pruned_node.py

```diff
@@ -39,12 +39,12 @@ def test_pruned_node(cronos):
         CONTRACTS["TestERC20A"],
         key=KEYS["validator"],
     )
-    tx = erc20.functions.transfer(ADDRS["community"], 10).buildTransaction(
+    tx = erc20.functions.transfer(ADDRS["community"], 10).build_transaction(
         {"from": ADDRS["validator"]}
     )
     signed = sign_transaction(w3, tx, KEYS["validator"])
     txhash = w3.eth.send_raw_transaction(signed.rawTransaction)
-    exp_gas_used = 51520
+    exp_gas_used = 51503
 
```

Comment:
gas change related to [fill_transaction_defaults](https://github.com/ethereum/web3.py/commit/b28c5da6579f95e63a99bbdf1d0605b37d241921)

Reply:
i see 👍